src/main/resources/META-INF/plugin.xml

```diff
@@ -29,6 +29,10 @@
 
     <actions>
 
+<!--        is it searchable? should not be -->
```

Comment:
Hmmm I only see
![image](https://user-images.githubusercontent.com/3383210/193421825-4895d59a-89ce-4e8e-954c-edc4442aeb32.png)

Reply:
So probably **not** searchable for whatever reason